plugins/link/link.cpp

```diff
@@ -31,8 +31,8 @@ struct LinkedMem {
 	wchar_t description[2048];
 };
 
-static void about(HWND h) {
```

Comment:
I'd have preferred a uintptr_t or so, but void * works -- it's practically the only type we can guarantee is pointer-sized on Windows w/o including other headers like stdint.h (which, in the past, have been hard to come by on Windows...).

Anyway, I'll change it to void *.